jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -110,6 +110,14 @@
          description: Number of compute nodes
 
      - string:
+         name: nodenumberlonelynode
```

Comment:
Can we please skip this switch and use the `$cinder_backend` variable to compute if this is needed (if set to "nfs") and set `$nodenumberlonelynode=1` from within the mkcloud job.

Background: the setting of nodes and the configuring of the different types is about to change soon. So I don't want to lead people to using this parameter now.